src/Containers/Calendar/Containers/Calendar/Calendar.spec.tsx

```diff
@@ -0,0 +1,53 @@
+import { render, waitForElement, fireEvent } from "@testing-library/react-native";
+import * as React from "react";
+import Calendar from "./Calendar";
+
+it("should render Week Calendar", async () => {
+  const onDateChange = jest.fn();
+  const { queryByText } = render(<Calendar currentDate={new Date("04/21/2020")} onDateChange={onDateChange} />);
+  const firstDay = await waitForElement(() => queryByText("Su"));
+  expect(firstDay).toBeTruthy();
+  const lastDay = await waitForElement(() => queryByText("Sa"));
+  expect(lastDay).toBeTruthy();
+});
+
+it("should toggle Month Calendar", async () => {
+  const onDateChange = jest.fn();
+  const { getByLabelText, getByText } = render(
+    <Calendar currentDate={new Date("04/21/2020")} onDateChange={onDateChange} />,
+  );
+  const ToggleMonthButton = getByLabelText("toggleMonthCalendar");
+  fireEvent.press(ToggleMonthButton);
+  const firstDay = await waitForElement(() => getByText("1"));
+  expect(firstDay).toBeTruthy();
```

Comment:
There is no need to use `expect` here. `waitForElement` (or `findByText`) will throw error, if no element has been found